includes/compats/woo-blocks/class-wc-amazon-payments-advanced-block-compat-classic.php

```diff
@@ -57,8 +57,9 @@ public function get_payment_method_data() {
 	 * @return array Return an array of script handles that have been registered already.
 	 */
 	protected function scripts_name_per_type( $type = '' ) {
-		$script_data = include wc_apa()->path . '/build/index.asset.php';
-		wp_register_script( 'amazon_payments_advanced_classic_block_compat', wc_apa()->plugin_url . '/build/index.js', $script_data['dependencies'], $script_data['version'], true );
+		$script_data = include wc_apa()->path . '/build/payments-methods/classic/index.asset.php';
```

Comment:
Build folder is being created by executing `npm run build`. Its not present in the repo, but its present in the zip being produced.

In this PR we also introduce a change in the .gitattributes file, `src export-ignore` so that the src directory will be excluded by the built version.

Package @wordpress/scripts, which we use for building scripts and js related to the Gateways integration with blocks, also produce a file {finename}.asset.php which contains information about the scripts dependencies and version.

Reply:
Gotcha, thanks for clarifying!